src/vcpkg/commands.export.cpp

```diff
@@ -479,15 +420,17 @@ namespace
                 const InstallDir dirs =
                     InstallDir::from_destination_root(export_paths, action.spec.triplet(), binary_paragraph);
 
+                // +1 for the trailing slash
+                auto triplet_prefix_size = action.spec.triplet().to_string().size() + 1;
                 auto lines =
                     fs.read_lines(paths.installed().listfile_path(binary_paragraph)).value_or_exit(VCPKG_LINE_INFO);
                 std::vector<Path> files;
                 for (auto&& suffix : lines)
                 {
-                    if (suffix.empty()) continue;
                     if (suffix.back() == '/') suffix.pop_back();
```

Comment:
Does this crash if there's an empty line in the list file? It seems like this should switch places with the next line.